opm/input/eclipse/Parser/Parser.hpp

```diff
@@ -97,7 +97,8 @@ namespace Opm {
         bool hasKeyword( const std::string& ) const;
         const ParserKeyword& getKeyword(const std::string& name) const;
 
-        bool isRecognizedKeyword( const std::string_view& deckKeywordName) const;
+        bool isRecognizedKeyword(std::string_view deckKeywordName) const;
+        bool isBaseRecognizedKeyword(std::string_view deckKeywordName) const;
```

Comment:
Would be cool to add some documentation here to outline the difference between these two functions